extensions/src/main/java/org/apache/falcon/extensions/store/ExtensionStore.java

```diff
@@ -72,6 +87,32 @@ private ExtensionStore() {
         }
         storePath = new Path(uri);
         fs = initializeFileSystem();
+        initializeDbTable();
+    }
+
+    private void initializeDbTable() {
+        try{
+            metricStore.deleteTrustedExtensionMetadata(TRUSTED_EXTENSION);
+            List<String> extensions = getExtensions();
+            for (String extension : extensions) {
+                String extensionType = AbstractExtension.isExtensionTrusted(extension) ? TRUSTED_EXTENSION
+                        : CUSTOM_EXTENSION;
+                String description = getShortDescription(extension);
+                String recipeName = extension;
+                String location = storePath.toString() + '/' + extension;
+                metricStore.storeExtensionMetadataBean(recipeName, location, extensionType, description);
+            }
+        } catch (FalconException e){
+            LOG.error("Exception in ExtensionStore:", e);
+            throw new RuntimeException(e);
+        }
+
+    }
+
+    private String getShortDescription(final String extensionName) throws FalconException {
```

Comment:
This can emulate what is done in org.apache.falcon.resource.extensions.ExtensionManager#getShortDescription. That way AbstractExtension does not have to have a getDescription method.